src/routes/console/organization-[organization]/settings/deleteOrganizationModal.svelte

```diff
@@ -37,6 +37,8 @@
                         : $organizationList.teams[0].$id;
 
                 await goto(`${base}/console/organization-${teamId}`);
+            } else if ($organizationList?.total == 1) {
+                await goto(`${base}/console/organization-${$organizationList.teams[0].$id}`);
```

Comment:
I would avoid this change, I don't think the design team would approve. 
A developer might be deleting a project with the intention of creating a new one, simply trying to wipe it and re-create it from appwrite.json, for instance.

Redirecting him to a project would stand in the way.

In other scenarios, it could also cause confusion, making the developer think it's same project, just dashboard. He would think something went wrong, will refresh page and try to delete again - causing him to delete a whole different project.

Reply:
I'd see an onboarding page every time on an org. deletion, thought it wasn't expected if there's already atleast one organisation available. But the reasoning regarding the confusion makes sense. I'll roll back this change.

Reply:
Done.